include/chowdsp_wdf/rtype/rtype_detail.h

```diff
@@ -67,7 +67,11 @@ namespace wdft
                 v_type bc {};
                 for (int r = 0; r < vec_size; r += simd_size)
                     bc = xsimd::fma (xsimd::load_aligned (S_[c].data() + r), xsimd::load_aligned (a_ + r), bc);
-                b_[c] = xsimd::hadd (bc);
+                #if XSIMD_VERSION_MAJOR > 8
```

Comment:
Awesome thanks! I usually try to do `VERSION >= 9`, since I find it a little easier to understand, but either way is fine. It would also be cool to leave a comment mentioning that XSIMD changed the name of the method in version 9.0.

Looks like the CI is failing because of a formatting thing, but if you run `clang-format` that should fix itself.

Finally, if you could add version 9.0.0 to the CI tests [here](https://github.com/Chowdhury-DSP/chowdsp_wdf/blob/main/.github/workflows/test.yml#L21), that would be super helpful!